sources/install/package_web.sh

```diff
@@ -40,48 +50,60 @@ function install_whatweb() {
 }
 
 function install_wfuzz() {
+    # CODE-CHECK-WHITELIST=add-aliases
     colorecho "Installing wfuzz"
     apt --purge remove python3-pycurl -y
     fapt libcurl4-openssl-dev libssl-dev
-    python3 -m pip install pycurl wfuzz
+    pip3 install pycurl wfuzz
+    mkdir /usr/share/wfuzz
+    git -C /tmp clone --depth 1 https://github.com/xmendez/wfuzz.git
+    mv /tmp/wfuzz/wordlist/* /usr/share/wfuzz
+    rm -rf /tmp/wfuzz
     add-history wfuzz
     add-test-command "wfuzz --help"
+    add-test-command "[[ -d '/usr/share/wfuzz/' ]] || exit 1"
```

Comment:
Quick tips: when you when to do this kind of basic test with an inline command use the command `test` instead of the bracket `[[ ]]`

Reply:
Fixing